Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM jinaai/jina:2.0
+RUN apt-get update && apt-get install -y python3.7 python3.7-dev python3-pip git
```

Comment:
```suggestion
FROM jinaai/jina:2.0.7-py37-standard

RUN apt-get update && apt-get install --no-install-recommends -y git \
                && rm -rf /var/lib/apt/lists/*
```

Reply:
This needs to be done with automation, not manually. Needs to be discussed